go/pkg/bertymessenger/service.go

```diff
@@ -632,6 +632,12 @@ func (svc *service) sendAccountUserInfo(ctx context.Context, groupPK string) (er
 		})...)
 	}
 
+	// return if there is no user info
+	if acc.GetDisplayName() == "" && avatarCID == "" {
```

Comment:
we should not return here, the user might want to remove it's display name or avatar

and if we really want this it should return before re-encrypting the avatar imo

Reply:
Ok I understand, but currently if there is no display name AND no avatar, the payload in the metadata store is empty, and `handleAppMessageSetUserInfo` returns an error (go/internal/messengerpayloads/handlers.go:941).
What do you advise me?